### ISSUE TITLE
abi.encodeMethod.encode biginteger issue

### ISSUE THREAD

**Issue by leoaion** [aionnetwork/aion_web3#84]
**Web3 version**: aion-web3 v1.2.6-beta.0

**Command(s) triggering the issue**:
```
abi.encodeMethod(
                    'undelegate', 
                    ['BigInteger'], 
                    [new BigNumber('10000000000000000000').toFixed()])
```

**API output**: 
the encoded value of 10*10^18 is 2308 8ac7230489e80000. it's not 2's complement representation which misses sign(+/-).
The expected value is 2309 008ac7230489e80000.

**Comment by K-White** [comment]
a fix for this is available as of npm 1.2.6-beta.6 release

**Comment by aionfarbod** [comment]
@K-White  can we close this issue?
